msf_core/src/lib/gps_conversion.cc

```diff
@@ -21,14 +21,18 @@
 
 namespace msf_core {
 
-GPSConversion::GPSConversion() {
+GPSConversion::GPSConversion()
+{
```

Comment:
This is not our convention. Please revert. Are you using the correct formatter? https://github.com/ethz-asl/programming_guidelines/wiki/Code-formatter